services/core/java/com/android/server/clipboard/ClipboardService.java

```diff
@@ -62,6 +62,7 @@
 import android.os.UserManager;
 import android.provider.DeviceConfig;
 import android.provider.Settings;
+import android.ext.settings.ExtSettings;
```

Comment:
Imports in majority of AOSP code are ordered alphabetically ascending (A to Z) from left to right, could this be moved to be right above `import android.graphics.drawable.Drawable;`?

```
import android.content.pm.UserInfo;
import android.ext.settings.ExtSettings;
import android.graphics.drawable.Drawable;
```